Packs/AzureSentinel/Integrations/AzureSentinel/README.md

```diff
@@ -124,6 +124,7 @@ Gets a list of incidents from Azure Sentinel.
 ##### Base Command
 
 `azure-sentinel-list-incidents`
+`azure-sentinel-update-incident incident_id=1234567 severity=Informational assignee_email=example@example.com`
```

Comment:
this should not appear here - this only the base command.
you might want to generate a new command example and put its inputs below.

Reply:
thanks